pkg/cmd/customrun/delete.go

```diff
@@ -100,14 +88,14 @@ func deleteCustomRuns(s *cli.Stream, p cli.Params, crNames []string) error {
 	namespace := p.Namespace()
 	for _, crName := range crNames {
 		// Check if CustomRun exists before attempting deletion
-		exists, _ := crExists([]string{crName}, p)
-		if len(exists) == 0 {
+		err := crExists(crName, p)
```

Comment:
as we are creating client before, can we pass `client` and `namespace`, instead of `p`